rpc_jobs/irr_role_test.yml

```diff
@@ -0,0 +1,102 @@
+- project:
```

Comment:
Would it make sense to have a repo axis? Then a couple of  jobs can be generated per repo, which will keep the histories independent. It looks like this JJB project is aimed at generating a set of jobs, but that one set of jobs will deal with every IRR?